packages/protobuf-bench/README.md

```diff
@@ -10,5 +10,5 @@ server would usually do.
 
 | code generator      | bundle size             | minified               | compressed         |
 |---------------------|------------------------:|-----------------------:|-------------------:|
-| protobuf-es         | 74,055 b      | 36,960 b | 9,651 b |
+| protobuf-es         | 74,495 b      | 36,880 b | 9,644 b |
```

Comment:
I think this is caused by `packages/protobuf-bench/node_modules/@bufbuild/protobuf` in the lock file. It should be `node_modules/@bufbuild/protobuf`.

I guess the import paths make their way into the bundle, increasing the size.

Perhaps @smaye81 can lend you a hand?

Reply:
@timostamm @seankimdev Should be good now.  I basically just checked out the package-lock from main and reran `make`.  Looks like the benchmark values are back to being unchanged.

Reply:
Thanks Steve!